lib/api/dsl/storage/index.js

```diff
@@ -150,7 +171,7 @@ class Storage {
    * @return {object}
    */
   store (index, collection, filters) {
-    let result = addFilter(this.hashKey, this.filters, index, collection, filters);
+    const result = addFilter(this.hashKey, this.filters, index, collection, filters);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 85 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)